test/gurka/test_build_admin.cc

```diff
@@ -284,4 +395,36 @@ TEST(AdminTest, TestBuildAdminFromPBF) {
     EXPECT_EQ(X_admin.state_text(), "Hyogo");
     EXPECT_EQ(X_admin.country_text(), "Japan");
   }
+
+  // check a and c are in Germany
+  {
+    auto a_id = findNode(graph_reader, admin_map.nodes, "a");
+    EXPECT_TRUE(a_id.Is_Valid());
+    auto c_id = findNode(graph_reader, admin_map.nodes, "c");
+    EXPECT_TRUE(c_id.Is_Valid());
+    const auto* a = graph_reader.nodeinfo(a_id);
+    EXPECT_EQ(a->drive_on_right(), true);
+    EXPECT_EQ(a->named_intersection(), false);
+    const auto* c = graph_reader.nodeinfo(c_id);
+    EXPECT_EQ(c->drive_on_right(), true);
+    EXPECT_EQ(c->named_intersection(), false);
+    AdminInfo a_admin = graph_reader.GetGraphTile(a_id)->admininfo(a->admin_index());
+    EXPECT_EQ(a_admin.state_text(), "");
+    EXPECT_EQ(a_admin.country_text(), "Germany");
+    AdminInfo c_admin = graph_reader.GetGraphTile(c_id)->admininfo(c->admin_index());
+    EXPECT_EQ(c_admin.state_text(), "");
+    EXPECT_EQ(c_admin.country_text(), "Germany");
+  }
+
+  // b is in an austrian enclave
+  {
+    auto b_id = findNode(graph_reader, admin_map.nodes, "b");
+    EXPECT_TRUE(b_id.Is_Valid());
+    const auto* b = graph_reader.nodeinfo(b_id);
+    EXPECT_EQ(b->drive_on_right(), true);
+    EXPECT_EQ(b->named_intersection(), false);
+    AdminInfo b_admin = graph_reader.GetGraphTile(b_id)->admininfo(b->admin_index());
+    EXPECT_EQ(b_admin.state_text(), "");
+    EXPECT_EQ(b_admin.country_text(), "Austria");
+  }
```

Comment:
so i added way more than just germany and austria in the map some of the thigns i added are errornous on purpose so the code handles them robustly (so we cant check for their presence because they get ignored). a few others check for very special geometric issues (self intersections etc) which would abort the program so no tests here for that either. i could just throw in a some basic checks here though to make sure those admins made it through. ill do it a bit later.